src/Game.js

```diff
@@ -0,0 +1,81 @@
+var Game = function() {
+
+  this.rolls = [];
+  this.firstRoll = true;
+  this.firstRollScore = 0;
+
+};
+
+Game.prototype.roll = function(pins) {
+  var game = this;
+  pinsError();
+  this.rolls.push(pins);
+  rollOrder();
+
+  function rollOrder() {
+    if (game.firstRoll == true && pins < 10) {
+      game.firstRoll = false;
+      game.firstRollScore = pins;
+    } else {
+      game.firstRoll = true;
+      game.firstRollScore = 0;
+    }    
+  };
+
+  function pinsError() {
+    var a = "Max second roll score is between 0 and "
+    var b = (10 - game.firstRollScore).toString();
+    if (pins < 0) {
+      throw new Error("Min roll score is 0")
+    } else if (game.firstRoll == true && pins > 10) {
+      throw new Error("Max first roll score is 10")
+    } else if (game.firstRoll == false && pins > (10 - game.firstRollScore)) {
+      throw new Error(a += b)
+    }
+
+  };
+
+};
+
+
+
+
+Game.prototype.score = function() {
+  
+  var total = 0;
+  var i = 0;
+  var game = this;
+
+  for (f = 0; f < 10; f++) {
```

Comment:
You're forgetting a var keyword - now you have a global `f`!
Use i for for loop interation (at least for the first loop) - or if you want to be really descriptive, you could use use `var frame`